products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/settings/{{cookiecutter.environment_name}}/{{cookiecutter.segment_name}}/jenkins/asFile/site.properties

```diff
@@ -1,8 +1,8 @@
-AUTOMATION_BASE_DIR=/opt/codeontap/automation
-AUTOMATION_DIR=/opt/codeontap/automation/jenkins/aws
-GENERATION_BASE_DIR=/opt/codeontap/generation
-GENERATION_STARTUP_DIR=/opt/codeontap/startup
-GENERATION_DIR=/opt/codeontap/generation/aws
+AUTOMATION_BASE_DIR=/opt/hamlet/automation
+AUTOMATION_DIR=/opt/hamlet/automation/jenkins/aws
+GENERATION_BASE_DIR=/opt/hamlet/generation
+GENERATION_STARTUP_DIR=/opt/hamlet/startup
+GENERATION_DIR=/opt/hamlet/generation/aws
```

Comment:
These should probably be updated to align with what we have in the hamlet container I don't think they are quite right there

Reply:
updated here: https://github.com/hamlet-io/executor-cookiecutter/pull/8/commits/7befc4b32aef36c535ae9cfaed1a4798038d3d00